packages/test/test-drivers/src/odspTestDriver.ts

```diff
@@ -390,4 +395,81 @@ export class OdspTestDriver implements ITestDriver {
             dataStorePath: "/",
         });
     }
+
+    // Executes the API call to set sensitivity labels on file with provided itemID
+    public static async setSensitivityLabel(siteUrl: string, itemId: string, labelId: string, loginConfig: IOdspTestLoginInfo)    {
```

Comment:
Why static? It might be better to use api was being used by other methods.

Reply:
- Added the abstraction currently in place for API as pointed out.
- Static because the function requires only the site url and item ID so doesn't require the testDriver object. Took inspiration from some of previous functions like getStorageToken which had similar requirements and are defined as static.

Reply:
Done